mycroft/skills/main.py

```diff
@@ -133,9 +133,13 @@ def check_connection():
             # Skip the sync message when unpaired because the prompt to go to
             # home.mycrof.ai will be displayed by the pairing skill
             enclosure.mouth_text(mycroft.dialog.get("message_synching.clock"))
+
         # Force a sync of the local clock with the internet
-        ws.emit(Message("system.ntp.sync"))
-        time.sleep(15)   # TODO: Generate/listen for a message response...
+        config = Configuration.get()
+        platform = config.get("enclosure", {}).get("platform", "unknown")
```

Comment:
We can assume `"enclosure"` exists in the config since it is written in `mycroft/configuration/mycroft.conf`, so this can be written as `config["enclosure"].get(...`.